src/main/java/core/basesyntax/service/FruitShopService.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitRecordDto;
+import java.util.List;
+import java.util.Map;
+
+public interface FruitShopService {
+    boolean applySuitableOperation(List<FruitRecordDto> fruitRecordDtos);
```

Comment:
This method saves data to db, so rename it to save and return void no need to return true all the time. You can check later if you data was added to db or not

Reply:
done
I'll leave the methods like that for now. I specially made them return values. And when I write unit-tests, I will refactor